project/app/api/predict.py

```diff
@@ -161,6 +174,24 @@ async def predict_gas(item: GasItem):
 
     return round(total, 2)
 
+@router.post('/predict/airbnb')
+async def predict_airbnb(item: AirbnbItem):
+    """
+    Predicts the total cost of the airbnb for x nights for a length of stay.
+    
+    Params:
+        Inputs: Integer of destination's longitude, 
+                Integer of destination's latitude,
+                Integer of Number of nights stay
+
+        Outputs: A float with the total cost of the airbnb cost for the length of the stay
+    """
+    lat = item.Airbnb_lat
+    long = item.Airbnb_long
+    nights = item.Airbnb_nights
+    
+    result = AIRBNB_MODEL.predict([[lat, long, nights]])[0]
```

Comment:
This line might throw a 500 without the model.  We could comment it out, and put in an example prediction.